src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -93,7 +93,6 @@ internal MethodInfo ReplaceWithExtensionImplementation(out bool wasError)
 
                 wasError = (Method is not null && method is null) || (SetMethod is not null && setMethod is null);
 
-                // Tracked by https://github.com/dotnet/roslyn/issues/76130 : Test with indexers (ie. "method") and in compound assignment (ie. "setMethod")
```

Comment:
>and in compound assignment (ie. "setMethod") [](http://example.com/codeflow?start=112&length=44)

Is the "and in compound assignment (ie. "setMethod")" part covered or tracked somewhere?

Reply:
Yes, the follow-up note was moved to https://github.com/dotnet/roslyn/issues/78829

Reply:
> Yes, the follow-up note was moved to ...

The issue looks indexer specific, but the mentioned scenarios is not.